src/main/scala/zio/json/encoder.scala

```diff
@@ -4,42 +4,75 @@ import scala.annotation._
 import scala.collection.mutable
 import scala.collection.immutable
 
-// convenient `.toJson` syntax
-object syntax {
-  implicit final class EncoderOps[A](private val a: A) extends AnyVal {
-    def toJson(implicit A: Encoder[A]): String = A.toJson(a, None)
+import zio.{ Chunk, Managed, Queue, UIO, ZIO, ZManaged }
+import zio.blocking._
+import zio.stream._
 
-    // Jon Pretty's better looking brother, but a bit slower
```

Comment:
!!

Reply:
It still exists, it just got moved to the package object!